__tests__/contact/contact.mutation.test.js

```diff
@@ -1,215 +1,196 @@
-                openHours {
-                  lang
-                  value
-                }
-                address {
-                  lang
-                  value
-                }
-                email
-                images {
-                  value {
-                    medium
-                  }
-                }
-                link
+  beforeAll(async () => {
+    operations = await setupApp();
+  });
+
+  it('#1 should add contact to database', async () => {
+    const res = await operations.mutate({
+      mutation: gql`
+        mutation($contact: contactInput!) {
+          addContact(contact: $contact) {
+            ... on Contact {
+              _id
+              phoneNumber
+              openHours {
+                lang
+                value
+              }
+              address {
+                lang
+                value
               }
-              ... on Error {
-                message
-                statusCode
+              email
+              images {
+                value {
+                  medium
+                }
               }
+              link
+            }
+            ... on Error {
+              message
+              statusCode
             }
           }
-        `,
-        variables: { contact },
-      })
-      .then(response => response)
-      .catch(e => e);
+        }
+      `,
+      variables: { contact },
+    });
+
     contactsId = res.data.addContact._id;
-    expect(res.data.addContact).toHaveProperty('phoneNumber', '1241241242144');
-    expect(res.data.addContact).toHaveProperty('openHours', [
-      {
-        __typename: 'Language',
-        lang: contact.openHours[0].lang,
-        value: contact.openHours[0].value,
-      },
-      {
-        __typename: 'Language',
-        lang: contact.openHours[1].lang,
-        value: contact.openHours[1].value,
-      },
-    ]);
-    expect(res.data.addContact.openHours).toBeInstanceOf(Array);
-    expect(res.data.addContact).toHaveProperty('address', [
-      {
-        __typename: 'Language',
-        lang: contact.address[0].lang,
-        value: contact.address[0].value,
-      },
-      {
-        __typename: 'Language',
-        lang: contact.address[1].lang,
-        value: contact.address[1].value,
-      },
-    ]);
-    expect(res.data.addContact.address).toBeInstanceOf(Object);
-    expect(res.data.addContact).toHaveProperty('email', newContact.email);
-    expect(res.data.addContact.images).toBeInstanceOf(Array);
-    expect(res.data.addContact).toHaveProperty('images', [
+    const addedContact = res.data.addContact;
+
+    expect(addedContact).toHaveProperty('phoneNumber', contact.phoneNumber);
+    expect(addedContact).toHaveProperty(
+      'openHours',
+      contact.openHours.map(item => ({
+        ...item,
+      }))
+    );
+    expect(addedContact.openHours).toBeInstanceOf(Array);
+    expect(addedContact).toHaveProperty(
+      'address',
+      contact.address.map(item => ({
```

Comment:
Please, simplify it to the `contact.address` because mapping is useless now due to the returning the same object